app.cr

```diff
@@ -9,13 +9,20 @@ def headers
   headers
 end
 
-def crystal_repos
+def crystal_repos(sort)
   client = HTTP::Client.new("api.github.com", 443, true)
-  response = client.get("/search/repositories?q=language:crystal&per_page=100", headers)
+  response = client.get("/search/repositories?q=language:crystal&per_page=100&sort=#{sort}", headers)
```

Comment:
We can use [redis](https://github.com/stefanwille/crystal-redis), heroku supports it out of the box and it's free. What do you think?

I can do this in a separate pull request, if you want, or you can do it yourself, whichever you prefer.